apps/dashboard/app/(app)/team-switcher.tsx

```diff
@@ -22,7 +22,12 @@ import { useOrganization, useOrganizationList, useUser } from "@clerk/nextjs";
 import { ScrollArea } from "@/components/ui/scroll-area";
 import Link from "next/link";
 
-export const WorkspaceSwitcher: React.FC = (): JSX.Element => {
+type Props = {
+  workspace: {
+    name: string;
+  };
+};
+export const WorkspaceSwitcher: React.FC<Props> = (props): JSX.Element => {
```

Comment:
`export const WorkspaceSwitcher: React.FC<Props> = (props): JSX.Element => `

It's a nitpick but maybe just this 👇🏻 ?
` export const WorkspaceSwitcher = (props: Props) => `